metaflow/plugins/kfp/tests/sample_flows/nested_foreach.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from metaflow import FlowSpec, step, Parameter, JSONType
+import pprint
+
+
+def truncate(var):
+    var = str(var)
+    if len(var) > 500:
+        var = "%s..." % var[:500]
+    return var
+
+
+class ExpectationFailed(Exception):
+    def __init__(self, expected, got):
+        super(ExpectationFailed, self).__init__(
+            "Expected result: %s, got %s" % (truncate(expected), truncate(got))
+        )
+
+
+def assert_equals(expected, got):
+    if expected != got:
+        raise ExpectationFailed(expected, got)
+
+
+class NestedForeach(FlowSpec):
```

Comment:
please add
`""" """` comments, 
the
```
    """
    foreach -> split -> linear -> foreach -> linear -> join
    """
```
style comments are very useful, thanks!